### PR TITLE
docs: Fix Frame Processor install step and QualityPrioritization Order

### PR DESCRIPTION
**Pull request by astanb**
fix: switched incorrect property ordering for `qualityPrioritization` options
fix: added extra step required for creating a frame processing plugin on Android

I couldn't work out how to test the changes using docosaurus, but hopefully they're so small that they're ok!

### CODE REVIEW

**Comment by mrousavy** [comment]
Hey! Thank you for your PR!

You can test those changes by running:

```
cd docs
yarn start
```

or 

```
cd docs
yarn build
yarn serve
```

but the CI GitHub action also tests the docs build :)

**Comment by astanb** [comment]
Ah thanks, I actually had to add in a `yarn` before `yarn start` to install all the dependencies for docosaurus.

I've also updated the contributing guide with that info to help others who might want to update the docs.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -43,6 +43,13 @@ Read the READMEs in [`android/`](android/README.md) and [`ios/`](ios/README.md)
 
 > Run `yarn check-android` to validate codestyle
 
+### Docs
+
+1. Edit the relevant file, it may be easiest to search for what you're editing to find the right file
+2. Install the documentation dependencies using `yarn` or `npm install` from the `docs` folder.
```
Comment (mrousavy) [inline CONTRIBUTING.md]:
```suggestion
2. Install all dependencies by running `yarn` inside the `docs` folder.
```

**Review by mrousavy — CHANGES_REQUESTED** [review]
thanks for the guide, please update the minor change and we're good to go

**Comment by astanb** [comment]
All done :white_check_mark:

**Pull request merged**